src/server/dflycmd.cc

```diff
@@ -40,6 +40,8 @@ ABSL_RETIRED_FLAG(uint32_t, allow_partial_sync_with_lsn_diff, 0,
 ABSL_DECLARE_FLAG(bool, info_replication_valkey_compatible);
 ABSL_DECLARE_FLAG(uint32_t, replication_timeout);
 ABSL_DECLARE_FLAG(uint32_t, shard_repl_backlog_len);
+ABSL_FLAG(bool, experimental_force_takeover, false,
```

Comment:
@romange @dranikpg 

Maybe instead introduce a subcommand to takeover, like `force` instead of a flag ? That way if we have a failed Takeover we can follow up with Forced one ?